extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -285,12 +290,51 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this.resourceTypeService.startDeployment(this.getCurrentProvider());
 	}
 
-	private installTools(): void {
-		this._dialogObject.message = {
-			level: azdata.window.MessageLevel.Information,
-			text: localize('deploymentDialog.InstallingTools', "Required tools are being installed now.")
-		};
+	private UpdateToolsTableData(tool: ITool) {
+		this._toolsTable.data = this._toolsTable.data.map(rowData => {
+			if (rowData[0] === tool.displayName) {
```

Comment:
Not sure why you're creating a new set of data - why not just modify the existing data?

Reply:
fixed

Reply:
Undid my fix. The underlying azdata.window.Dialog does not repaint the html table if only individual cells or just individual rows in the data table are changed. The whole data needs to be udpated for it to repain.